vehicles_dealership/__manifest__.py

```diff
@@ -4,7 +4,7 @@
     # Module Information
     "name": "Vehicles Dealership",
     "category": "vehicles",
-    "version": "16.0.1.0.0",
+    "version": "17.0.1.0.0",
```

Comment:
Add author, website information, This is our application and contribution so that information should have to add in manifest.